tests/tests.js

```diff
@@ -0,0 +1,693 @@
+var chai = require('chai');
+var usngs = require('../usng');
+var converter = new usngs.Converter();
+
```

Comment:
You should add tests for points and bounding boxes going both directions for every hemisphere too.